spirekey/src/app/(wallet)/accounts/[caccount]/devices/[cid]/send/page.tsx

```diff
@@ -1,18 +1,11 @@
 'use client';
 
-import { Box } from '@kadena/kode-ui';
 import dynamic from 'next/dynamic';
-import React from 'react';
 
 const SendForm = dynamic(() => import('@/components/SendForm/SendForm'), {
   ssr: false,
 });
 
 export default function SendPage() {
-  return (
-    <Box margin="lg">
-      <h1>Send</h1>
-      <SendForm />
-    </Box>
-  );
+  return <SendForm />;
```

Comment:
Can't we move this logic to the higher component as it's not doing a lot anymore?